Solutions/Digital Shadows/Data Connectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -18,22 +18,30 @@ def __init__(self, id, key, secret, url):
         self.id = id
```

Comment:
in the supplied requirements.txt can we remove urllib please? It is part of core python and can't be separately installed.

Reply:
Removing this from requirements causes an urllib3 missing error when deployed. We have seen this quite a lot when developing and testing. Not really sure why it happens.

Reply:
`urllib3` is different to `urllib` - you can put `urllib3` in a requirements file

Reply:
@shikhin-metron - could you try `urllib3` instead of `urllib` please and let me know if that works fine when deployed?